Spigot-Server-Patches/0319-Add-more-Zombie-API.patch

```diff
@@ -5,18 +5,48 @@ Subject: [PATCH] Add more Zombie API
 
 
 diff --git a/src/main/java/net/minecraft/server/EntityZombie.java b/src/main/java/net/minecraft/server/EntityZombie.java
-index 12f954723ac2f0058081d810ce83aed99532caad..02a845f99bb445fb4aa66a90273469a848195747 100644
+index 12f954723ac2f0058081d810ce83aed99532caad..b72f42e5355e392dbec59e7827f746fffc22f466 100644
 --- a/src/main/java/net/minecraft/server/EntityZombie.java
 +++ b/src/main/java/net/minecraft/server/EntityZombie.java
-@@ -33,6 +33,7 @@ public class EntityZombie extends EntityMonster {
+@@ -1,21 +1,22 @@
+ package net.minecraft.server;
+ 
+ import com.mojang.serialization.DynamicOps;
++import org.bukkit.entity.Zombie;
++import org.bukkit.event.entity.CreatureSpawnEvent;
++import org.bukkit.event.entity.EntityCombustByEntityEvent;
++import org.bukkit.event.entity.EntityTargetEvent;
++import org.bukkit.event.entity.EntityTransformEvent;
++
++import javax.annotation.Nullable;
++
+ import java.time.LocalDate;
+ import java.time.temporal.ChronoField;
+ import java.util.List;
+ import java.util.Random;
+ import java.util.UUID;
+ import java.util.function.Predicate;
+-import javax.annotation.Nullable;
+ 
+ // CraftBukkit start
+-import org.bukkit.craftbukkit.event.CraftEventFactory;
+-import org.bukkit.entity.Zombie;
+-import org.bukkit.event.entity.CreatureSpawnEvent;
+-import org.bukkit.event.entity.EntityCombustByEntityEvent;
+-import org.bukkit.event.entity.EntityTargetEvent;
+-import org.bukkit.event.entity.EntityTransformEvent;
```

Comment:
Disable automatic import sorting in your IDE - you shouldn't touch these.

Reply:
Thank you for the feedback! I replaced this MR with https://github.com/PaperMC/Paper/pull/4271 (as I did not know how to fix it in the existing branch).

Reply:
You just need to repeat steps you did, apart from creating new branch.
You'll then have 2 commits with changes, that GH will automatically squash for viewing - and later they'll be squashed by dev team when merging to master.